funnel/models/proposal.py

```diff
@@ -231,6 +232,15 @@ def votes_by_date(self):
                     votes_bydate[groupname][date] += -1 if vote.votedown else +1
         return votes_bydate
 
+    def is_outstation_speaker(self, space_location_geonameid):
```

Comment:
You could now change this to `is_matching_geonameid` and accept a `geonameid` parameter. Makes this a bit more generic.